src/wallet/walletdb.h

```diff
@@ -101,6 +101,7 @@ class CHDChain
     }
 
     void IncrementAccountCounter() {
+        // TODO: We should check for overflow somewhere and handle it.
```

Comment:
This overflow is actually reachable because it's only at 2<sup>31</sup>. Please file an issue.

Reply:
Opened #5487.